### ISSUE TITLE
Provision billing.uiowa.edu

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#381]
billing.uiowa.edu was provisioned recently in D7.

They haven't started creating content yet so I talked with them and they would like us to provision this site out in D8.

I don't think a cert was ever created that covers billing.uiowa.edu. We will need to add this once the site is provisioned in D8. We will also need to deprovision the D7 site once the D8 site is ready.

**Comment by briand44** [comment]
This has been provisioned out, users added and communicated. Customer may want some custom development related to redirects but will open a new issue if that is the case.
